src/SartAlgorithm.cpp

```diff
@@ -146,8 +145,8 @@ bool CSartAlgorithm::initialize(const Config& _cfg)
 		}
 		for (int i = 0; i < projectionCount-1; i++) {
 			int k = (rand() % (projectionCount - i));
-			int t = projectionOrder[i];
-			m_piProjectionOrder[i] = projectionOrder[i + k];
+			int t = m_piProjectionOrder[i];
+			m_piProjectionOrder[i] = m_piProjectionOrder[i + k];
```

Comment:
Thanks for catching that! It looks like I was confused by near similarities with `CudaSartAlgorithm` here...